editor/src/clj/editor/prefs.clj

```diff
@@ -75,19 +78,20 @@
    :properties
    {:asset-browser {:type :object
                     :properties
-                    {:track-active-tab {:type :boolean :label "Track Active Tab in Asset Browser"}}}
+                    {:track-active-tab {:type :boolean
+                                        :ui {:label "Track Active Tab in Asset Browser"}}}}
```

Comment:
I kind of feel like the `:ui` clause could be warranted to keep on its own line like this everywhere.